db/migration/src/m20240226_000001_create_channel.rs

```diff
@@ -0,0 +1,96 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(Channel::Table)
+                    .if_not_exists()
+                    .col(
+                        ColumnDef::new(Channel::Id)
```

Comment:
Why are we using integer ids here instead of the actual channel ids as primary keys?

Reply:
Primary keys should always be autoincrement Integers. We can then have another IDs which will have index on top.